Dockerfile

```diff
@@ -28,5 +34,7 @@ RUN apt-get -qqy update && apt-get install -qqy \
         kubectl && \
     gcloud config set core/disable_usage_reporting true && \
     gcloud config set component_manager/disable_update_check true && \
-    gcloud config set metrics/environment github_docker_image
+    gcloud config set metrics/environment github_docker_image && \
+    CLOUDSDK_COMPONENT_MANAGER_DISABLE_UPDATE_CHECK=true gcloud --version && \
```

Comment:
let's remove these --version checks, we can add separately. I assume they are for validating stuff, but they only validate _some_ stuff.